tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1113,3 +1113,14 @@ test_i2c_mmc56x3: mmc56x3@92 {
 	magn-odr = <0>;
 	auto-self-reset;
 };
+
+test_i2c_tmp1075: tmp1075@91 {
```

Comment:
The address should be 0x98. The previous node is incorrect and should be 0x97, but that can be fixed in a separate patch.

Reply:
updated both addresses in a separate commit